ljclang_Index_h.lua

```diff
@@ -1000,10 +1055,13 @@ enum CXTemplateArgumentKind {
  unsigned clang_isFunctionTypeVariadic(CXType T);
  CXType clang_getCursorResultType(CXCursor C);
  unsigned clang_isPODType(CXType T);
+ unsigned clang_isFinalType(CXType T);
```

Comment:
Curious. Generating that file from Clang 3.9's libclang (using the one from http://apt.llvm.org/ for Ubuntu 16.04) does not give these two functions, and also not `clang_CXX_isDeleted` or `clang_CXXMethod_isOverride`. Was that with a local build on your side after this 3.9 release? I guess I might pull 4.0 for further development then.